frontend/src/components/Notifications.svelte

```diff
@@ -75,6 +78,11 @@
     export function hide() {
         shown = false
     }
+
+    export function setToastTimeout(timeout) {
```

Comment:
I don't see this new function being utilized, if it isn't just delete it.

Reply:
Sorry about that, removed.